src/BehaviorsSDKNative/NativeUnitTests/NativeUnitTests.nuget.props

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition=" '$(ExcludeRestorePackageImports)' != 'true' ">
+    <RestoreSuccess Condition=" '$(RestoreSuccess)' == '' ">True</RestoreSuccess>
+    <RestoreTool Condition=" '$(RestoreTool)' == '' ">NuGet</RestoreTool>
+    <ProjectAssetsFile Condition=" '$(ProjectAssetsFile)' == '' ">$(MSBuildThisFileDirectory)project.lock.json</ProjectAssetsFile>
+    <NuGetPackageRoot Condition=" '$(NuGetPackageRoot)' == '' ">$(UserProfile)\.nuget\packages\</NuGetPackageRoot>
+    <NuGetPackageFolders Condition=" '$(NuGetPackageFolders)' == '' ">C:\Users\DAVAUGH\.nuget\packages\;C:\Microsoft\Xamarin\NuGet\</NuGetPackageFolders>
```

Comment:
>C:\Users\DAVAUGH [](start = 70, length = 16)

A lot more of these :-)

Reply:
I've added an ignore to .gitignore for these NuGet v3 related files. I've also removed the nuget.props files from the PR.